postory/android/Postory/app/src/test/java/com/example/postory/PostModelConverterTest.java

```diff
@@ -0,0 +1,112 @@
+package com.example.postory;
+
+import static org.junit.Assert.assertEquals;
+
+import com.example.postory.models.OtherUser;
+import com.example.postory.models.Post;
+import com.example.postory.models.UserGeneralModel;
+import com.example.postory.models.UserModel;
+import com.example.postory.utils.UserModelConverter;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+public class PostModelConverterTest {
+
+    Post modelGround = new Post();
+    Post modelParsed = new Post();
+
+    String model1Json = "{\"id\":9,\"title\":\"Goat in Ölüdeniz\",\"story\":\"Seeing a goat near the beach eating some bread pieces was really a unique experience for me.\",\"owner\":1,\"username\":\"ahmetkus\",\"tags\":[\"fun\",\"vacation\",\"summer\"],\"locations\":[[\"Ölüdeniz\",36.67366496090383,29.102678261697296]],\"year\":[2020,2020],\"month\":[6,7],\"day\":[],\"hour\":[],\"minute\":[],\"images\":[\"https://group9-451-project.s3.amazonaws.com/image/IMG_20210710_201947.jpg\"],\"postDate\":\"2022-01-04T09:45:27.687000Z\",\"editDate\":\"2022-01-04T09:45:27.687000Z\",\"viewCount\":0,\"comments\":[],\"likeList\":[[6.0,\"mstfec\"]],\"userPhoto\":\"\"}";
```

Comment:
Using json strings is a good way of initializing objects with many fields. Nice!